docs/packages.md

```diff
@@ -132,9 +132,59 @@ Example:
+![Create a package](assets/pics/qs-file-packages/create-file.png)
+
+4. Select the package owner from the dropdown.
+![Select package owner](assets/pics/qs-file-packages/file-package-select-owner.png)
+
+5. Give your package a name.
+![Name your package](assets/pics/qs-file-packages/file-package-name.png)
+
+### Adding an Incoming URL
+This section explains what the Incoming URLs are and how to use a URL template for Event Collection. 
+
+1.) Add the URL path where your events will be collected. This is the user visible endpoint your application will connect to for event submission. This setting while required is not relevant when submitting events via the [Event Import API](https://docs.scarf.sh/event-import/).
+ > Note: You can use a URL template, but if you use variables in your URL they need to also be used in your Incoming Path that is defined in the next step.
+
+![path where files are located](assets/pics/qs-file-packages/file-package-outgoing-url.png)
+
+2.) Choose the domain where your events will be submitted. You may choose to use your own domain or you may choose to use `<username>.gateway.scarf.sh` provided by default by Scarf.
+
+3.) Click **Submit**.
+
+### Configuring Event Collection
+Once an Event Collection package has been created, you are ready to collect [Custom telemetry](https://docs.scarf.sh/custom-telemetry/)
+
+## Python Packages
+
+Scarf Gateway configuration for a Python package entry has three main considerations:
+
+- pip Command: This is the current pip command used to install your package. For packages on PyPI.org, this will be of the form pip install my-pkg and will include the --extra-index-url https://my-python-project-domain.com if your package is hosted elsewhere. This defines the location where the users will be redirected to when installing your package.
+- Domain: This can be your own domain, or a Scarf-supplied domain, of the form <username>.gateway.scarf.sh. By default, your Scarf domain will be used if this field is left empty.
+- Telemetry: This allows you to gather insights into how your package is used without collecting any personally identifiable information.
+
+Installing Python packages via requirements.txt
+Add the --extra-index-url option at the top of your requirements.txt:
+
+(''' --extra-index-url https://my-python-project-domain.com/simple/ ''')
+my-pkg==0.0.1
+
+NOTE: We have noticed indeterminate behavior in some versions of Pip that have resulted in the public registry being used for download regardless of the --extra-index-url addition.
+
+If you elect to use your own domain, you'll need to add a CNAME for that domain to <span style="color:gray;">https://gateway.scarf.sh</span>. Additionally we require you to verify your ownership of the domain by setting a TXT with a value that Scarf provides upon package creation. See your DNS provider's instructions for how to add CNAME and TXT records.
```

Comment:
Use the ` character to do inline highlighting, no need to make html spans directly.